website/src/content/docs/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Welcome to jscodeshift
+description: Get started with building and running jscodeshift codemods.
+template: splash
+hero:
+  tagline: Get started with building and running jscodeshift codemods.
+  image:
+    file: ../../assets/houston.webp
```

Comment:
Wonder if we can put a more relevant image here

Reply:
I'm definitely planning on doing that. I'll see if we can have a simple logo + illustration for jscodeshift to use elsewhere on the website as well.

Reply:
I switched the index from `splash` to `docs` template and removed this illustration so we can publish the docs for now. I'll update this later with a relevant image.